azext_hanaonazure/custom.py

```diff
@@ -6,6 +6,28 @@
 # pylint: disable=no-self-use,too-many-lines
 
 
+def create_hanainstance(client, location, resource_group_name, instance_name, ssh_public_key, os_computer_name, ip_address):
+    try:
```

Comment:
Excuse my limited knowledge on this extension codebase. Are we trying to import from python3 campatible version first and if we fail then we fall back to 2.7?

Reply:
Correct